docs/descriptors/terraform_tflint.md

```diff
@@ -18,6 +18,8 @@ Set variable TERRAFORM_TFLINT_SECURED_ENV to `false` if you need `tflint --init`
 - See [How to disable tflint rules in files](https://github.com/terraform-linters/tflint/blob/master/docs/guides/annotations.md){target=_blank}
 - See [Index of problems detected by tflint](https://github.com/terraform-linters/tflint/tree/master/docs/rules#rules){target=_blank}
 
+> If you are using the GitHub action please use the `TERRAFORM_TFLINT_UNSECURED_ENV_VARIABLES: GITHUB_TOKEN` to prevent plugin download issues
```

Comment:
@ruzickap this page is generated from terraform.megalinter-descriptor.yml, but it's ok, I'll merge it in one of my PRs then fix it then declare you as co-author :)